data/presets/shop/variety_store.json

```diff
@@ -7,5 +7,13 @@
     "tags": {
         "shop": "variety_store"
     },
-    "name": "Variety Store"
+    "terms": [
+        "discount",
+        "five and dime",
+        "five and ten"
+    ],
+    "name": "Variety Store",
+    "aliases": [
+        "Dollar Store"
```

Comment:
“Variety Store” [shouldn’t be the main name](https://books.google.com/ngrams/graph?content=variety+store%2Cdollar+store%2Cdiscount+store&year_start=1800&year_end=2019&corpus=28&smoothing=3&direct_url=t1%3B%2Cvariety%20store%3B%2Cc0%3B.t1%3B%2Cdollar%20store%3B%2Cc0%3B.t1%3B%2Cdiscount%20store%3B%2Cc0). “Discount Store” has been more common for years, though “Dollar Store” has recently surpassed it in usage.